### ISSUE TITLE
[Question] Will you add harfbuzz ?

### ISSUE THREAD

**Issue by pureexe** [amarullz/AROMA-Filemanager#8]
Thai language have problem to display
but i think will work fine with harfbuzz :)
Please try.
Best regards.

**Comment by amarullz** [comment]
yes. harfbuzz-ng will be used in libaroma. I need to port many code for next major version. still long release date.

**Comment by pureexe** [comment]
Thank you for reply.
I will waiting for you.
